go-controller/pkg/ovn/master_test.go

```diff
@@ -1678,6 +1679,103 @@ var _ = ginkgo.Describe("Default network controller operations", func() {
 		})
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	})
+
+	ginkgo.It("verify IPs in host-network-namespace address_set after remote zone node add or delete", func() {
+		app.Action = func(ctx *cli.Context) error {
+			_, err := config.InitConfig(ctx, nil, nil)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			config.OVNKubernetesFeature.EnableInterconnect = true
+			config.Kubernetes.HostNetworkNamespace = "ovn-host-network"
+
+			// PortBinding is required to add interconect resources for
+			// remote zone nodes
+			pb := &sbdb.PortBinding{
```

Comment:
but do you need to manually add the portBinding in SBDB along with transit switch? is it the same logic as "unable to start with empty nodeList?" if so please combine the comments at L1690 and L1701 saying "we need to start the DB setup here with PB and transit switch since we want to simulate remote node add and thus we start with empty nodeList which is not a real life scenario. This is only for UT test simulation

Reply:
As per discussion: We need to create port bindings on behalf of ovn-controller so that the node gets added successfully.